install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -1,7 +1,84 @@
-# Use --set to specify the Pilot and Policy IPs on the main Istio control plane
+# Use --set to specify the endpoints of the main Istio control plane
+# NB: These defaults do not work properly - they must be overriden with
+# IP addresses
+
 global:
   # The Istio control plane Pilot endpoint
-  pilotEndpoint: none
+  pilotEndpoint: istio-pilot.istio-system
 
   # The Istio control plane Policy endpoint
-  policyEndpoint: none
+  policyEndpoint: istio-policy.istio-system
+
+  # The Istio control plane statsd endpoint
+  statsdEndpoint: istio-statsd-prom-bridge.istio-system
+
+  # Default tag for Istio images.
+  hub: docker.io/istionightly
+
+  # Default tag for Istio images.
+  # Should track latest released version.
+  # Currently using nightly build, for testing
+  tag: circleci-nightly
+  proxy:
+    image: proxy
+
+  # imagePullPolicy is applied to istio control plane components.
+  imagePullPolicy: IfNotPresent
+
+  # Not recommended for user to configure this. Hyperkube image to use when creating custom resources
+  hyperkube:
+    repository: quay.io/coreos/hyperkube
+    tag: v1.7.6_coreos.0
+
+  # Install istio CA.
+  securityEnabled: true
+
+  # controlPlaneMtls enabled. Will result in delays starting the pods while secrets are
+  # propagated, not recommended for tests.
+  controlPlaneSecurityEnabled: false
+
+  # Default mtls policy. If true, mtls between services will be enabled by default.
+  mtls:
+    # Default setting for service-to-service mtls. Can be set explicitly using
+    # destination rules or service annotations.
+    enabled: false
+    # List of fully qualified services to exclude from mtls
+    # TODO: add the templating.
+    mtlsExcludedServices:
+    - "kubernetes.default.svc.cluster.local"
+
+  # create RBAC resources. Must be set for any cluster configured with rbac.
+  rbacEnabled: true
+
+  ## imagePullSecrets for all ServiceAccount. Must be set for any clustser configured with privte docker registry.
+  # imagePullSecrets:
+  #   - name: "private-registry-key"
+
+  # Default is 1 second
+  refreshInterval: 10s
+
+  # Specify pod scheduling arch(amd64, ppc64le, s390x) and weight as follows:
```

Comment:
Do we have this in the main install ? Don't remember...

Reply:
indeed.